index.d.ts

```diff
@@ -1,6 +1,12 @@
 import {BrowserWindow, DownloadItem} from 'electron';
 
 declare namespace electronDl {
+	interface Progress {
+		percent: number;
+		transferred: number;
```

Comment:
Transferred *what*? I think we should name it `transferredBytes` for clarity. Same with the below.

Reply:
changed 👌